Alignment/HIPAlignmentAlgorithm/plugins/LhcTrackAnalyzer.cc

```diff
@@ -69,7 +69,7 @@ class LhcTrackAnalyzer : public edm::one::EDAnalyzer<> {
   //=======================
   void SetVarToZero();
 
-  static const int nMaxtracks_ = 3000;
+  static constexpr int nMaxtracks_ = 3000;
```

Comment:
out of curiosity, this code has been like that since ages, see e.g.:

https://github.com/cms-sw/cmssw/blob/5c0525f1f7a145b6567d7a81736bf66b90508d1d/Alignment/HIPAlignmentAlgorithm/interface/LhcTrackAnalyzer.h#L77

why wasn't it giving issues before?

Reply:
previously this code was in the shared library but now it is part of plugin and edm plugins are build with `-fvisibility-inlines-hidden` may be this the reason for this failure